From e1a573bae4534b7fc2c1f08c3dff967321b3e041 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Sun, 11 Sep 2016 15:46:30 +0900
Subject: [PATCH] deal with GPG2

gpg1 and 2 output different secret key information, and now Debian moves to
gpg2 for default. It's neccesary to update it to deal with it.

$ gpg1 -K
/home/henrich/.gnupg/secring.gpg
--------------------------------
sec   1024D/C4C7264B 2004-06-11
(snip)

$ gpg2 -K
/home/henrich/.gnupg/pubring.gpg
--------------------------------
sec   dsa1024 2004-06-11 [SC]
      455582ED38B6C870E099388C22ED21CBC4C7264B
(snip)
---
 scripts/debsign.bash_completion | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/debsign.bash_completion b/scripts/debsign.bash_completion
index bcc1063..2c42ee0 100644
--- a/scripts/debsign.bash_completion
+++ b/scripts/debsign.bash_completion
@@ -14,7 +14,7 @@ _debsign()
 
     case $prev in
         -k)
-            keyid_options=`gpg -K|grep ^sec|cut -d'/' -f2|cut -d' ' -f1`
+            keyid_options=`gpg -K|grep ^' '|cut -d' ' -f7`
             COMPREPLY=( $( compgen -W "$keyid_options" | grep ^$cur ) )
             ;;
         *)
-- 
2.9.3

